cumulusci/robotframework/Salesforce.robot

```diff
@@ -1,7 +1,9 @@
 *** Settings ***
 
 Library        Collections
+Library        OperatingSystem
 Library        String
+Library        XML
```

Comment:
Why the new libraries here?

Reply:
The goal of Salesforce.robot is to provide the most common utilities needed to write Salesforce robot tests.  As I found in creating the DF demo, the OperatingSystem and XML libraries are quite helpful, especially for integration tests so it seems to make sense to include them by default.  They're both standard libraries in robot so I think the impact would be minimal.

Reply:
That makes sense. Please mention it for the release notes.